packages/sdk/src/__tests__/contract-first-example-smoke.test.ts

```
@@ -0,0 +1,48 @@
+
+import { Workflow } from "../workflow.js";
+
+describe("contract-first example smoke", () => {
+  it("loads the canonical contract-first example and preserves its public authoring surface", async () => {
+    const workflowPath = join(
+      process.cwd(),
+      "..",
+      "..",
+      "examples",
+      "07-contract-first-evaluation",
+      "workflow.yaml",
```

ESLint

[error] Delete `,`

(prettier/prettier)